tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -326,31 +326,59 @@ def _prepare_bitstream_firmware(self, config):
 
     @staticmethod
     def get_cpu_extension(cpu_extensions, selection_mode):
+        def get_cpu_extensions_list(file_format, base_name, selection_mode='',):
```

Comment:
The default value is never used here, so it's a bit pointless.

Reply:
fixed